frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -130,18 +130,48 @@ describe("scenarios > question > settings", () => {
       cy.findByText("Visible columns").click();
       findColumnAtIndex("Products → Category", 5);
 
+      // https://github.com/metabase/metabase/pull/21338#pullrequestreview-928807257
+
+      // Add "Address"
+      cy.findByText("Address")
+        .siblings(".Icon-add")
+        .click();
       /**
        * Helper functions related to THIS test only
        */
 
+      // The result automatically load when adding new fields
+      cy.wait("@dataset");
+
+      // Refresh @sidebarColumns as we added a new field
+      cy.findByText("Click and drag to change their order")
+        .parent()
+        .find(".cursor-grab")
+        .as("sidebarColumns"); // Store all columns in an array
+
+      findColumnAtIndex("User → Address", 16).as("user-address");
```

Comment:
FYI, you can also use index `-1` for the last element, and `-2` for the second to last element if you don't care exactly how many elements are in the array.

Reply:
Deeyum! That's neat. I'll do that, as that tells the story better.